Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -18,34 +18,48 @@
 namespace ROS2
 {
     void CollidersMaker::AddColliders(urdf::LinkSharedPtr link, AZ::EntityId entityId)
-    {
-        if (link->collision_array.size() == 0)
-        { // one or zero colliders - element is used
-            AddCollider(link->collision, entityId);
-            return;
+    { // TODO - refactor out naming (colliders, visuals)
+        auto colliderName = AZStd::string::format("%s_collider", link->name.c_str());
+        int nameSuffixIndex = 1; // For disambiguation when multiple unnamed colliders are present. The order does not matter here
```

Comment:
Nitpick: if it is an index, it would be better to start counting from `0`.

Reply:
This is a displayed number, so indexing for human-readable names should start with 1
(we say - the first item, the second item)